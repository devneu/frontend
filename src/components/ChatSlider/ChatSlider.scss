.chat-slider {

  .slider-img {
    max-width: 27px;
    height: 27px;
    border-radius: 100%;
    border: 1px solid rgb(252, 253, 255);
    margin: 3px;
    background: #0050b3;
  }

  .slider {
    margin-right: 10px;
    max-height: 150px;
    max-width: 100px;

  }

  .slick-prev:before, .slick-next:before {
    color: black;
  }

  .slick-prev {
    top: 74px;
    left: 30px;
    padding-top: 10px;
    height: 20px;
    width: 12px;
  }

  .slick-next {
    top: 74px;
    left: 70px;
    padding-top: 10px;
    height: 20px;
    width: 12px;
  }

  .slides-number {
    text-align: center;
    font-size: 16px;
  }

}
