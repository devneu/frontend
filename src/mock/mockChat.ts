export const photos = [
  {
    img: 'https://images.unsplash.com/photo-1469854523086-cc02fe5d8800?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8M3x8fGVufDB8fHx8&w=1000&q=80',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSV_EfZmXNnuKrtvDecfz6B7r955J6X7EFTsw&usqp=CAU',
  },
  {
    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrOELmQBbTGeOYYBHmT6UxFVma2GOPE-6kpg&usqp=CAU',
  },
  {
    img: 'https://sourceduk.com/wp-content/uploads/2018/10/girlgoneabroad.jpg',
  },
  {
    img: 'https://www.workandtravelgroup.com/wp-content/uploads/wtg-group-2.jpg',
  },
  {
    img: 'https://www.moneycrashers.com/wp-content/uploads/2018/10/travel-world-map-1068x713.jpg',
  },
];

export const attachments = [
  {
    name: 'pdf',
    img: 'https://www.iconpacks.net/icons/2/free-pdf-icon-3385.png',
    description: 'how to create a case study',
  },
  {
    name: 'zip',
    img: 'https://www.iconpacks.net/icons/2/free-zip-icon-1519-thumb.png',
    description: 'Project Documents',
  },
  {
    name: 'word',
    img: 'https://www.pngkey.com/png/full/210-2104640_imagen-microsoft-office-365-word-logo.png ',
    description: 'Competitor Analysis Template',
  },
];

export const avatar = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
};

export const groupData = {
  img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
  name: 'Nasir Uddin',
  description: 'Ux Designer',
  icon: 'https://i.pinimg.com/originals/44/9d/d1/449dd1f38a8bd87b41f7b960db0fefff.png',
  memberImgs: [
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar2.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/howto/img_avatar.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU',
    },
    {
      img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU',
    },
    {
      img: 'https://www.w3schools.com/w3images/avatar6.png',
    },
  ],
};
