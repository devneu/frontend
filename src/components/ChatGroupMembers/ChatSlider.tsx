import React, { useState } from 'react';
import Slider, { CustomArrowProps, Settings } from 'react-slick';
import 'slick-carousel/slick/slick.scss';
import 'slick-carousel/slick/slick-theme.scss';
import { Avatar } from 'antd';
import { GroupChat } from '../../types/ChatPreview.interface';
import './ChatSlider.scss';

interface ChatSliderProps {
  slides: GroupChat[];
}

const ChatSlider = ({ slides }: ChatSliderProps) => {
  const [activeSlideNumber, setActiveSlideNumber] = useState(1);
  const SLIDER_ITEMS_NUMBER = slides.length;
  const ITEMS_PER_SLIDE = 6;
  const lastSlideNumber = Math.ceil(SLIDER_ITEMS_NUMBER / ITEMS_PER_SLIDE);

  const SliderArrowNext = ({ currentSlide, slideCount, ...props }: CustomArrowProps) => (
    <img src={`${process.env.PUBLIC_URL}/images/next-arrow.png`} alt="prevArrow" {...props} />
  );
  const SliderArrowPrev = ({ currentSlide, slideCount, ...props }: CustomArrowProps) => (
    <img src={`${process.env.PUBLIC_URL}/images/prev-arrow.png`} alt="prevArrow" {...props} />
  );
  const settings: Settings = {
    className: 'slider',
    arrows: true,
    infinite: true,
    slidesToShow: 1,
    speed: 500,
    rows: 2,
    slidesPerRow: 3,
    nextArrow: <SliderArrowNext currentSlide={0} slideCount={lastSlideNumber} />,
    prevArrow: <SliderArrowPrev currentSlide={0} slideCount={lastSlideNumber} />,
    // eslint-disable-next-line @typescript-eslint/no-unused-vars-experimental
    beforeChange: (current: number, next: number) => {
      setActiveSlideNumber(next + 1);
    },
  };

  return (
    <div className="chat-slider">
      <Slider {...settings}>
        {slides.map((item) => (
          <Avatar className="slider-img" alt="avatar" src={item.img} key={item.img} />
        ))}
      </Slider>
      <div className="slides-number">
        {activeSlideNumber}
        /
        {lastSlideNumber}
      </div>
    </div>
  );
};
export default ChatSlider;
