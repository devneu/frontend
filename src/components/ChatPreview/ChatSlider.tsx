import React from 'react';
import Slider from 'react-slick';
import 'slick-carousel/slick/slick.scss';
import 'slick-carousel/slick/slick-theme.scss';
import './ChatSlider.scss';
import { Avatar } from 'antd';
import { ChatData } from '../../types/ChatPreview.interface';

interface SliderProps {
  slider: ChatData[];
}
const ChatSlider = ({ slider }: SliderProps) => {
  const settings = {
    className: 'simpleSlider',
    arrows: true,
    infinite: true,
    slidesToShow: 1,
    speed: 500,
    rows: 2,
    slidesPerRow: 3,
  };
  return (
    <div className="simple-slider">
      <Slider {...settings}>
        {slider.map((item: any) => (
          <div>
            <Avatar className="slider-img" src={item.img} key={item.img} />
          </div>
        ))}
      </Slider>
    </div>
  );
};
export default ChatSlider;
