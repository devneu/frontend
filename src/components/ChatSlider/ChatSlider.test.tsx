import React from 'react';
import renderer from 'react-test-renderer';
import ChatSlider from './ChatSlider';
import { slides } from '../../mock/mockChat';

describe('ChatSlider', () => {
  it('Should successfully render', () => {
    const wrapper = renderer.create(<ChatSlider slides={slides} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
