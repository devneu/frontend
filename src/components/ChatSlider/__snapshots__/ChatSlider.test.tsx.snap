// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatSlider Should successfully render 1`] = `
<div
  className="chat-slider"
>
  <div
    className="slick-slider slider slick-initialized"
    dir="ltr"
  >
    <img
      alt="prevArrow"
      className="slick-arrow slick-prev"
      data-role="none"
      onClick={[Function]}
      src="/images/prev-arrow.png"
      style={
        Object {
          "display": "block",
        }
      }
    />
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/w3images/avatar6.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/howto/img_avatar2.png"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/howto/img_avatar.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/w3images/avatar6.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQnPGqX4s6HDBoVTLwIhy3fFmdxvMiDIfUtdA&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/howto/img_avatar2.png"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={4}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/howto/img_avatar.png"
              />
            </span>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={5}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://international-home.net/wp-content/uploads/2018/12/matureman1-512.png"
              />
            </span>
          </div>
          <div>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQVxhAxJ4D7MOeTTj6kR9PBeZonW5HM7giKjTbEmR-HMBwf3G1VqGnlwpO1kWrdyIZu8_U&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSgnq9pVEA16U0vH0nT0UeFY9vrTn99Za2a7QWub_dBpXSYTCZtBQULWaaRJ4ENFreEmPc&usqp=CAU"
              />
            </span>
            <span
              className="ant-avatar ant-avatar-circle ant-avatar-image slider-img"
              style={
                Object {
                  "display": "inline-block",
                  "width": "33.333333333333336%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt="avatar"
                onError={[Function]}
                src="https://www.w3schools.com/w3images/avatar6.png"
              />
            </span>
          </div>
        </div>
      </div>
    </div>
    <img
      alt="prevArrow"
      className="slick-arrow slick-next"
      data-role="none"
      onClick={[Function]}
      src="/images/next-arrow.png"
      style={
        Object {
          "display": "block",
        }
      }
    />
  </div>
  <div
    className="slides-number"
  >
    1
    /
    3
  </div>
</div>
`;
